doc/ref_model/chapters/chapter04.md

```diff
@@ -235,13 +235,15 @@ Table 4-9 shows capabilities related to resources allocation
 
 <a name="Table4-12"></a>
 
-| Ref       | Cloud Infrastructure management Capability | Unit   | Definition/Notes                                                 |
-|-----------|--------------------------------------------|--------|------------------------------------------------------------------|
-| e.man.001 | Virtual Compute allocation                 | Yes/No | Capability to allocate virtual compute resources to a workload   |
-| e.man.002 | Virtual Storage allocation                 | Yes/No | Capability to allocate virtual storage resources to a workload    |
-| e.man.003 | Virtual Networking resources allocation    | Yes/No | Capability to allocate virtual networking resources to a workload |
-| e.man.004 | Multi-tenant isolation                     | Yes/No | Capability to isolate resources between tenants                  |
-| e.man.005 | Images management                          | Yes/No | Capability to manage workload software images                    |
+| Ref       | Cloud Infrastructure management Capability | Unit            | Definition/Notes                                                 |
+|-----------|--------------------------------------------|-----------------|------------------------------------------------------------------|
+| e.man.001 | Virtual Compute allocation                 | Yes/No          | Capability to allocate virtual compute resources to a workload   |
+| e.man.002 | Virtual Storage allocation                 | Yes/No          | Capability to allocate virtual storage resources to a workload    |
+| e.man.003 | Virtual Networking resources allocation    | Yes/No          | Capability to allocate virtual networking resources to a workload |
+| e.man.004 | Multi-tenant isolation                     | Yes/No          | Capability to isolate resources between tenants                  |
+| e.man.005 | Images management                          | Yes/No          | Capability to manage workload software images                    |
+| e.man.010 | Compute Availability Zones                 | list of strings | The names of each Compute Availability Zone that was defined to separate failure domains |
```

Comment:
The definition/notes should reflect taht AZs are not only failure domains. Same comment for e.man.011

Reply:
This is intended to be a list fo the AZs that are failure domain separations; not a list of all AZs.  So, it would be possible to create AZs for other reasons, but if they are not separated from a failure domain standpoint, they wouldn't be listed here.

Reply:
@kedmison 
1. How would one obtain a list of AZs that are defined by separate failure domains? OpenStack can provide you a list of AZs. 

1. My concern here is that we not set expectations that may not be supported by operators. For example, when we say separate power supply (failure domain) is that from separate UPSs or ....?

Reply:
I was expecting this would be a documented list rather than a list pulled from an API, so that when using this cloud infrastructure, VNF vendors could understand, enumerate, and depend on the failure-domain based zones.

The list is not actually prescriptive in the types of failure domain separation either, as it is expected that operators may not be able to resolve to a common definition, nor would that definition be consistent in all locations.  I think it's best left to an operator to decide what failure types need to be considered when defining failure domains, and perhaps later revisions of this document may be able to somewhat normalize that definition over time.